src/I18n/TranslatorRegistry.php

```diff
@@ -201,6 +201,7 @@ public function get(string $name, ?string $locale = null): ?Translator
         $keyName = str_replace('/', '.', $name);
         $key = "translations.{$keyName}.{$locale}";
         $translator = $this->_cacher->get($key);
+        gc_collect_cycles();
```

Comment:
Should this be wrapped in a PHP version check? It seems the PHP bug has been fixed in PHP 8.1 https://github.com/php/php-src/commit/4a4ae45a0bcb82b01a4386433b2e4ee45862fc01

Reply:
We at least need a comment. This could probably go in a much better situation if we looked later.